changelogs/emissions-api-update-sites-filtering-12-12-2023.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Emissions API v1 Update
+type: added
+---
+
+Added new Endpoints to the Emissions API to enable filtering by assets, as well as getting emissions value for sites.
+
+Below is a list of all the new endpoints:
+
+> ➕ GET: /emissions/period
+
+[Get period](ref:getemissions)
+
+> ➕ POST:
```

Comment:
should we put a URL (such as: emissions/period) after each GET and POST?

Reply:
This has been added.